plugins/modules/dcnm_vrf.py

```diff
@@ -216,28 +215,29 @@
       vrf_id: 9008011
       vrf_template: Default_VRF_Universal
       vrf_extension_template: Default_VRF_Extension_Universal
+      vlan_id: 2000
```

Comment:
As discussed today let's add an action plugin to warn users if they specify the `vlan_id` under the `attach:` block and terminate the ansible task.